run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.CH4

```diff
@@ -119,76 +120,164 @@ VerboseOnCores:              root       # Accepted values: root all
 (((EMISSIONS
 
 #==============================================================================
-# --- Gridded EPA (Maasakkers et al., Environ. Sci. Technol., 2016) ---
+# --- Gridded GHGI v2 (Maasakkers et al., submitted to ES&T, 2023) ---
 #
 # NOTES:
+# - This is the main Gridded GHGI v2 dataset based off the US GHGI published in 2020
```

Comment:
@msulprizio: Question: the time cycle flags are now EFY instead of C, which will stop the run unless the simulation is within the 2012-2018 date range.  Is this the intended behavior now?  Just wanted to confirm.

Reply:
ah I see this is not the default option.  No problem then.